src/default-configuration.ts

```diff
@@ -11,4 +11,5 @@ export const defaultAppConfiguration: AppConfiguration = {
   loaderUrl: "https://cdn.fromdoppler.com/loader/v1/loader.js",
   unlayerProjectId: 32092,
   dopplerLegacyBaseUrl: "https://appint.fromdoppler.net",
+  useDummies: false,
```

Comment:
@Cromeror what do you think about making this dummy by default? in real environments this value will be always overridden by the configuration.

Reply:
We talked with @Cromeror and we will try with this value in true...